development/mimir-microservices-mode/dev.dockerfile

```
@@ -1,4 +1,6 @@
-FROM golang:1.20.5
+
+ARG BUILD_IMAGE=use-compose-up-to-build-image
```

Do we need the default value here? A comment might be clearer.
```suggestion
ARG BUILD_IMAGE # Use ./compose-up.sh to build this image.
```